client/src/components/ratingsReviews/RatingsAndReviews.jsx

```diff
@@ -38,7 +42,7 @@ class RatingsAndReviews extends React.Component {
     console.log('Getting Reviews');
     axios.get(`reviews?sort=${sort}&product_id=${id}`)
       .then(res => {
-        console.log('getReviews:', res);
+        console.log('AXIOS GET REVIEWS:', res);
```

Comment:
LOL. Trying to stand out in the crowded console?

Reply:
As best as I can lol